README.md

```diff
@@ -52,10 +51,19 @@ documentation][docs]
 
 [examples]: ./examples
 
+## Options
+
+- `onError` (function, default is undefined). Used to ensure that the function
+  does not throw. **Note:** When applied, the function will return `204`
```

Comment:
Would make this statement stronger. Clarify that when the option is used, the handler will always return 204 and that this can not be overridden in this case

Reply:
Tried to write it better now, if you have an idea that makes it even clearer, let me know! :)